app/controllers/saved_searches_controller.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
```

Comment:
Did the advanced search task create this controller? Can you do saved searches without user accounts?